src/router.js

```diff
@@ -10,18 +10,16 @@ export default function (app) {
         }
       }
     },
-    hooks: {
-      onRender: function (model) {
-        return app.view[model.router.match]
-      }
-    },
-    subscriptions: [
-      function (_, actions) {
+    subscriptions: {
+      "render": function (model, view) {
```

Comment:
I presume this is just for consistency, given that a common syntax for name spacing events is `namespace:event` which is an invalid key when not a string. Could be wrong though..